app/views/static/buttons/generic_button_component.html.haml

```diff
@@ -0,0 +1,27 @@
+.clearfix
```

Comment:
Well, we can keep writing new and new code, or we can actually make it work with the old one.

There's really nothing here that couldn't work with non-component forms, is there? That's the whole point of converting to components inside-out after all.